src/lib.rs

```diff
@@ -5,3 +5,7 @@ pub mod helpers;
 pub mod report;
 pub mod threshold;
 pub mod user;
+
+#[allow(unused_imports)]
+#[macro_use]
+extern crate lazy_static;
```

Comment:
You should be able to use use lazy_static in the test code and access the macro that way.